spec/services/calculators/employment_monthly_value_calculator_spec.rb

```diff
@@ -16,23 +16,12 @@
     end
 
     context "when there are employment payments" do
-      before do
-        _payment = create(
-          :employment_payment,
-          employment:,
-          date: Date.yesterday,
-          gross_income_monthly_equiv: 100,
-          national_insurance_monthly_equiv: 10,
-          tax_monthly_equiv: 20,
-        )
-        _recent_payment = create(
-          :employment_payment,
-          employment:,
-          date: Date.current,
-          gross_income_monthly_equiv: 500,
-          national_insurance_monthly_equiv: 20,
-          tax_monthly_equiv: 50,
-        )
+      let(:monthly_equiv_payment_data) { Utilities::EmploymentIncomeMonthlyEquivalentCalculator::MonthlyEquivPaymentData }
```

Comment:
I think maybe this is telling us that MonthlyEquivPaymentData shouldn't be scoped to the calculator? I'm not sure though, as it doesn't survive much beyond that layer